<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>3</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Organizations/Networks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshOrgsButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>3</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>3</y>
      <width>661</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Administration</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Organization name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="orgNameEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>123456789012345678901234567890</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Licensing status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="licenseStateLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;state&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="licenseDevicesBrowser">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>121</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Expiration date:</string>
      </property>
     </widget>
     <widget class="QLabel" name="licenseExpDateLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>40</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;mmm dd, yyyy ttt&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="adPushUpdatesButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Push Updates</string>
      </property>
     </widget>
     <widget class="QPushButton" name="adSnapshotButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>510</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Take Snapshot</string>
      </property>
     </widget>
     <widget class="QTableView" name="orgInventoryView">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>451</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Inventory:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="snmpVerMenu">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>v2c</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>v3</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="snmpCheck">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>SNMP:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="snmpAuthMenu">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>280</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>MD5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SHA</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>280</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Auth password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="snmpAuthPassEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>280</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>123456789012345678901234567890</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="snmpAuthPassCheck">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>280</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>show</string>
      </property>
     </widget>
     <widget class="QComboBox" name="snmpPrivMenu">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>280</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>DES</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AES128</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>310</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Priv password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="snmpPrivPassEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>123456789012345678901234567890</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="snmpPrivPassCheck">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>310</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>show</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>340</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Peer IPs (allowed):</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="snmpPeerIPListBrowser">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>340</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="nonMVPNCheck">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Non-Meraki site-to-site VPN:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="nonMVPNPeersMenu">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>410</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="nonMVPNAddPeerButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>440</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Peer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nonMVPNPeerNameEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>440</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="placeholderText">
       <string>Non-Meraki VPN peer name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nonMVPNPeerIPEdit">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>410</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>100.100.100.100</string>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>410</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Public IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>440</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Secret:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nonMVPNSecretEdit">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>440</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>1234567890123456789012345</string>
      </property>
      <property name="maxLength">
       <number>25</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="nonMVPNSecretCheck">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>440</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>show</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>470</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Private subnets:</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="nonMVPNSubnetBrowser">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>470</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Network name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="networkNameEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>123456789012345678901234567890</string>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>MX</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>MS</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>MR</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>MV</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>MC</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Systems Manager</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>211</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
